tools/build/redo

```diff
@@ -266,6 +266,17 @@ Components = [
                   modes = 'clean',
                   gradle = ':core:invoker'),
 
+    makeComponent('lean-fresh',
```

Comment:
and this --- i think there's a way to add extra ansible parameters to the steps.
if not, i'd be ok removing this option entirely because it can be done from the command line already via `redo`... less is more.